demos/camera_demo/src/trilliumvideosource.cpp

```diff
@@ -43,56 +43,85 @@ TrilliumVideoSource::~TrilliumVideoSource()
 
 int TrilliumVideoSource::init()
```

Comment:
Should we send a command packet to move the camera to (0, 0) at the end of initialization? Does the video settings packet reset the position?

Reply:
Actually TrilliumControl is a better place to reset the control.